.github/workflows/ci.yml

```diff
@@ -36,7 +39,129 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Run tests
         uses: ./.github/actions/test
-      - name: Run examples
-        uses: ./.github/actions/examples
+        with:
+          disk_cache: test-${{ github.base_ref }}-${{ matrix.os }}-${{ matrix.python-version }}
+  benchmarks:
+    needs: tests
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v3
       - name: Run benchmarks
         uses: ./.github/actions/benchmarks
+        with:
+          disk_cache: benchmark-${{ github.base_ref }}-${{ matrix.os }}
+  wheel:
+    runs-on: ubuntu-20.04
+    needs: tests
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.8"
+      - name: Upload examples
+        uses: actions/upload-artifact@v3
+        with:
+          name: examples
+          path: examples/*
+      - name: build wheel
+        uses: ./.github/actions/wheel
+        with:
+          package_version: "0.1.1"
```

Comment:
This should not be a fixed value right?

Reply:
This is needed for E2E testing, not to release wheels. Release is handled by a separate `wheels` workflow, instead of an action here.